backend/package.json

```diff
@@ -3,19 +3,24 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "prisma": {
+    "seed": "ts-node prisma/seed.ts"
+  },
   "scripts": {
     "test": "jest",
     "clearJest": "jest --clearCache",
-    "dev": "concurrently \"npx tsc --watch\" \"nodemon -q dist/index.js\"",
-    "build": "tsc",
-    "start": "node dist/index.js"
+    "dev": "tsc && (concurrently \"tsc --watch\" \"tsc-alias -w\" \"nodemon -q dist/backend/index.js\")",
```

Comment:
is this not meant to be "dist/index.js\"

Reply:
I'm pretty sure its because we import the Zod schemas from the shared folder 'lib' into the backend, tsc creates an additional layer of directories in the dist folder
|-- dist
|   |-- backend
|   |   |-- index.js
|   |   |-- ...
|   |--  lib
|   |   |-- schemas.js